pymc3/sampling.py

```diff
@@ -218,11 +215,7 @@ def assign_step_methods(model, step=None, methods=STEP_METHODS, step_kwargs=None
 
 
 def _print_step_hierarchy(s, level=0):
-    if isinstance(s, (list, tuple)):
-        _log.info(">" * level + "list")
-        for i in s:
-            _print_step_hierarchy(i, level + 1)
```

Comment:
If `s` was a list, then lines 502-503 would already have taken care of that:
```python
    if isinstance(step, list):
        step = CompoundStep(step)
```

`s` can't be a tuple, however, as it's either the output of `assign_step_methods` (which can be either a list or a single step), or the output of `init_nuts`, which is a single step.

So, this seems to be unreachable

Reply:
well spotted!